crates/sui-framework/sources/governance/validator.move

```diff
@@ -741,7 +764,6 @@ module sui::validator {
         initial_stake_option: Option<Balance<SUI>>,
```

Comment:
We don't need the `initial_stake_option` here and in `new` anymore then now that we stake through the token allocation schedule.

Reply:
just pushed a commit for this.